efs.tf

```diff
@@ -0,0 +1,41 @@
+locals {
+  efs_name = "${var.name}-efs"
```

Comment:
No need to put `efs` in the `efs_name` is there?

Also I know I've been messing this up for a while, but greater specificity should be loaded on the front, not the back of the string.

Reply:
Also if I _were_ going to add `efs` to the name, I would do it in the parent module. This seems unnecessary

Reply:
```suggestion
```